core-library/src/main/java/org/silverpeas/core/index/indexing/parser/odf/OOParser.java

```diff
@@ -56,11 +56,6 @@
 @Named("openDocumentParser")
 public class OOParser implements Parser {
 
-  /**
-   * Constructor declaration
-   */
-  public OOParser() {
-  }
   private String tempFolder = null;
   private final String TMP_UNZIP_DIR = "tmpUnzipOpenOffice";
   private final Namespace NS_OO = Namespace.getNamespace("office",
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this field "NS_OO" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS00116)